decidim-admin/lib/decidim/admin/version.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Decidim
+  # This holds the decidim-admin version.
+  module Admin
+    def self.version
+      "0.7.0-pre"
```

Comment:
Would be nice to see if rails or rspec have some kind of script to make releases so we can adapt it to decidim 🤔